pkg/kritis/admission/admission_test.go

```diff
@@ -105,6 +105,9 @@ func Test_AdmissionResponse(t *testing.T) {
 		{"valid response when no review error", false, constants.SuccessStatus, constants.SuccessMessage},
 		{"valid response when review error", true, constants.FailureStatus, fmt.Sprintf("found violations in %s", testutil.QualifiedImage)},
 	}
+	mockGAP := func(namespace string) ([]kritisv1beta1.GenericAttestationPolicy, error) {
```

Comment:
This mock shouldn't be needed - can't grafeas be imported and spun up in memory only mode?

Reply:
I'm not sure I understand the suggestion. This is providing the Generic Attestation Policy, which is a Kritis-specific thing, unrelated to Grafeas.

Reply:
Ah got it - I thought it was stored in grafeas for some reason.